app/models/comment.rb

```diff
@@ -0,0 +1,4 @@
+class Comment < ApplicationRecord
+  belongs_to :user
+  belongs_to :post
+end
```

Comment:
- [ ] For your comment model, the requirements say that you need to create a method to increment the counter of the comments for the given post after you create a new one, so please make sure to work on that!